src/black/__init__.py

```diff
@@ -401,6 +411,7 @@ def main(
     fast: bool,
     pyi: bool,
     ipynb: bool,
+    python_cell_magics: List[str],
```

Comment:
```suggestion
    python_cell_magics: Sequence[str],
```
Seems like it's sometimes a tuple at runtime, which makes mypyc unhappy